src/libical/CMakeLists.txt

```
@@ -265,6 +265,8 @@ elseif(STATIC_ONLY)
   add_library(ical-static ALIAS ical)
 endif()
 
+target_include_directories(ical INTERFACE "$<INSTALL_INTERFACE:${INCLUDE_INSTALL_DIR}>")
```

do we expect people to do 
`#include <libical/ical.h>`

or

`#include <ical.h>` ?

If it's the latter this needs to be

```suggestion
target_include_directories(ical INTERFACE "$<INSTALL_INTERFACE:${INCLUDE_INSTALL_DIR}>/libical")
```